integrations/operator/Makefile

```diff
@@ -72,6 +72,9 @@ help: ## Display this help.
 
 ##@ Development
 
+.PHONY: crd ## Single command to generate anything CRD-related (manifests and docs)
+crd: crdgen crd-docs
```

Comment:
Oh, nice.

Please backport this so workflows don't change between branches.

Reply:
:+1: will backport the PR to every env

Reply:
Thanks!